libs/vkd3d/command.c

```diff
@@ -5399,7 +5445,7 @@ static void d3d12_command_list_copy_image(struct d3d12_command_list *list,
 
         VK_CALL(vkUpdateDescriptorSets(list->device->vk_device, 1, &vk_descriptor_write, 0, NULL));
 
-        VK_CALL(vkCmdBeginRenderPass(list->vk_command_buffer, &begin_info, VK_SUBPASS_CONTENTS_INLINE));
+        VK_CALL(vkCmdBeginRenderPass2KHR(list->vk_command_buffer, &begin_info, &subpass_begin_info));
```

Comment:
I really don't think we'll ever need this for image copies.

Reply:
I guess it's more for consistency.

Reply:
Yeah, mixing and matching renderpass and renderpass2 stuff is weird. Mesa and stuff must wrap this to that anyway so it avoids another  step.